Update session replay docs to point to the new SDK (#11255)
DataDog/documentation: content/en/real_user_monitoring/guide/session-replay-getting-started.md

  //  env: 'production',
  //  version: '1.0.0',
  sampleRate: 100,
  replaySampleRate: 100,
  trackInteractions: true,
})

DD_RUM.startSessionReplayRecording();
```

#### CDN
Replace the Browser SDK URL `https://www.datadoghq-browser-agent.com/datadog-rum.js` with `https://www.datadoghq-browser-agent.com/datadog-rum-v3.js`. When `DD_RUM.init()` is called, it also starts the Session Replay recording.

*Supported browsers*: The Session Replay recorder supports all the browsers supported by the RUM Browser SDK with the exception of IE11. See the [browser support table][3].

#### Configuration

The usual [RUM initialization parameters][4] are all supported. 

> The Session Replay does not start recording automatically when calling `init()`. To then start the recording, call `startSessionReplayRecording()`. This can be useful to conditionally start the recording (e.g, to only record authenticated user sessions):

Reviewer: ```suggestion
The Session Replay does not start recording automatically when calling `init()`. To start the recording, call `startSessionReplayRecording()`. This can be useful to conditionally start the recording, for example to only record authenticated user sessions:
```
